src/libraries/Descriptor.sol

```diff
@@ -402,6 +403,9 @@ library Descriptor {
     /// @param fee fee amount
     /// @return fee as a decimal string with percent sign
     function feeToPercentString(uint24 fee) internal pure returns (string memory) {
+        if (fee == LPFeeLibrary.DYNAMIC_FEE_FLAG) {
```

Comment:
should just be able to do `fee.isDynamicFee()` given you have the library imported